processing/src/main/java/org/apache/carbondata/processing/loading/parser/impl/JsonRowParser.java

```diff
@@ -56,6 +56,9 @@ public JsonRowParser(DataField[] dataFields) {
       Map<String, Object> jsonNodeMap =
           objectMapper.readValue(jsonString, new TypeReference<Map<String, Object>>() {
           });
+      if (jsonNodeMap == null) {
```

Comment:
Here `|| jsonNodeMap.size() == 0` check is not required?

Reply:
yeah, not required. removed it.